Homework_12/Task_1.py

```diff
@@ -0,0 +1,24 @@
+from sympy import *
+x = Symbol("x")
+fx = x**3 - 50*x
+gx = -x**4 + 88*x**2 - 241
+fgx = fx - gx
+
+roots = solve(fgx)
+roots
+
+cross_dict = {}
+c = 1 
+for z in roots:
+    cross_dict[c] = {"x": round(N(z),5), "y": round(N(-1*z**4 - 88*z**2-241),5)}
```

Comment:
Координаты точек пересечения найдены неверно. У вас минус перед 2-м слагаемым, а должен быть плюс. Но вы бы не ошиблись, если бы использовали функцию `gx`, которая у вас определена выше? а не определяли выражение заново
```suggestion
    cross_dict[c] = {"x": round(N(z),5), "y": round(N(gx.subs(x, z)),5)}
```